### PR TITLE
add mypy checks

### PR DESCRIPTION
**Pull request by blu-nitro**
add mypy checks and fix any problems mypy found

### CODE REVIEW

File: daemon/nextbox_daemon/worker.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
 from datetime import datetime as dt
+from typing import Union
 
 from nextbox_daemon.config import log, cfg
 from nextbox_daemon.status_board import board
 
 
 class BaseJob:
-    name = None
+    name: Union[str, None] = None
```
Comment (daringer) [inline daemon/nextbox_daemon/worker.py:16]: Please use `Optional[str]` here

Comment (blu-nitro) [reply daemon/nextbox_daemon/worker.py]: done

**Comment by blu-nitro** [comment]
@nitrokey-ci test

**Comment by robin-nitrokey** [comment]
@nitrokey-ci test

**Review by daringer — APPROVED** [review]

**Comment by blu-nitro** [comment]
@nitrokey-ci merge

**Pull request merged**